kie-server-parent/kie-server-wars/kie-server/src/main/ee7-resources/META-INF/kie-server-jms.xml

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<messaging-deployment xmlns="urn:jboss:messaging-activemq-deployment:1.0">
+  <server name="default">
+    <jms-destinations>
+
+      <!-- Kie Server REQUEST queue -->
+      <jms-queue name="KIE.SERVER.REQUEST">
+        <entry name="queue/KIE.SERVER.REQUEST" />
+        <entry name="java:jboss/exported/jms/queue/KIE.SERVER.REQUEST" />
+      </jms-queue>
+
+      <!-- Kie Server RESPONSE queue -->
+      <jms-queue name="KIE.SERVER.RESPONSE">
+        <entry name="queue/KIE.SERVER.RESPONSE" />
+        <entry name="java:jboss/exported/jms/queue/KIE.SERVER.RESPONSE" />
+      </jms-queue>
+
+      <!-- Kie Server EXECUTOR queue -->
+      <jms-queue name="KIE.SERVER.EXECUTOR">
+        <entry name="queue/KIE.SERVER.EXECUTOR" />
+      </jms-queue>
+
```

Comment:
could we include the commented part for easier enablement of external signals over JMS - jBPM specific: https://github.com/droolsjbpm/droolsjbpm-integration/blob/master/kie-server-parent/kie-server-wars/kie-server/src/main/shared-ee6-ee7-resources/META-INF/kie-server-jms.xml#L23-L30